python/lsst/pipe/supertask/configOverrides.py

```diff
@@ -62,6 +65,20 @@ def addValueOverride(self, field, value):
         """
         self._overrides += [('value', (field, value))]
 
+    def addNameTemplates(self, nameDictString):
+        """Add keys and values to be used in formatting config nameTemplates
```

Comment:
We should probably rename this (and the method on `PipelineTaskConfig`) to something more akin to the current command-line argument name.